Python 3.12 highlights invalid escape sequence in wmlxgettext dependency

```
Traceback (most recent call last):
  File "/tmp/wesnoth/data/tools/wmlxgettext", line 41, in <module>
    import pywmlx
  File "/tmp/wesnoth/data/tools/pywmlx/__init__.py", line 13, in <module>
    import pywmlx.state as statemachine
  File "/tmp/wesnoth/data/tools/pywmlx/state/__init__.py", line 1, in <module>
    from pywmlx.state.machine import setup
  File "/tmp/wesnoth/data/tools/pywmlx/state/machine.py", line 11, in <module>
    from pywmlx.state.wml_states import setup_wmlstates
  File "/tmp/wesnoth/data/tools/pywmlx/state/wml_states.py", line 27
    '''
    ^^^
SyntaxError: (value error) not enough values to unpack (expected 2, got 1)
```

A couple months ago there was a very similar report from Iris also for wmlxgettext and Python 3.12:
```
18:25:31 iris@mia ~/Projects/Project Ethea/Ethea-i18n git:master % make pot-update
    POT     wesnoth-Invasion_from_the_Unknown.pot
Traceback (most recent call last):
  File "/Users/iris/Local/bin/wmlxgettext", line 41, in <module>
    import pywmlx
  File "/Users/iris/Projects/wesnoth/data/tools/pywmlx/__init__.py", line 13, in <module>
    import pywmlx.state as statemachine
  File "/Users/iris/Projects/wesnoth/data/tools/pywmlx/state/__init__.py", line 1, in <module>
    from pywmlx.state.machine import setup
  File "/Users/iris/Projects/wesnoth/data/tools/pywmlx/state/machine.py", line 165
    value = re.sub('\\\s*$', '', value)
                   ^^^^^^^^
SyntaxError: (value error) not enough values to unpack (expected 2, got 1)
make: *** [Invasion_from_the_Unknown/wesnoth-Invasion_from_the_Unknown.pot] Error 1
```
which was fixed in 4be9aa85849010e93a9a3b0f0701d0630e7b9368.

--- a/data/tools/pywmlx/state/wml_states.py
+++ b/data/tools/pywmlx/state/wml_states.py
@@ -24,42 +24,6 @@ class WmlIdleState:
 
 
 
-'''
-class WmlDefineState:
-    def __init__(self):
-        self.regex = re.compile('\s*#(define|enddef|\s+wmlxgettext:\s+)', re.I)
-        self.iffail = 'wml_checkdom'
-
-    def run(self, xline, lineno, match):
-        if match.group(1).lower() == 'define':
-            # define
-            xline = None
-            if pywmlx.nodemanip.onDefineMacro is False:
-                pywmlx.nodemanip.onDefineMacro = True
-            else:
-                err_message = ("expected an #enddef before opening ANOTHER " +
-                               "macro definition with #define")
-                finfo = pywmlx.nodemanip.fileref + ":" + str(lineno)
-                wmlerr(finfo, err_message)
-        elif match.group(1).lower() == 'enddef':
-            # enddef
-            xline = None
-            if pywmlx.nodemanip.onDefineMacro is True:
-                pywmlx.nodemanip.onDefineMacro = False
-            else:
-                err_message = ("found an #enddef, but no macro definition " +
-                               "is pending. Perhaps you forgot to put a " +
-                               "#define somewhere?")
-                finfo = pywmlx.nodemanip.fileref + ":" + str(lineno)
-                wmlerr(finfo, err_message)
-        else:
-            # wmlxgettext: {WML CODE}
-            xline = xline [ match.end(): ]
-        return (xline, 'wml_idle')
-'''
-
-
-
 class WmlCheckdomState:
     def __init__(self):
         self.regex = re.compile(r'\s*#textdomain\s+(\S+)', re.I)

